include/mbedtls/config.h

```diff
@@ -654,6 +654,25 @@
  */
 //#define MBEDTLS_AES_SCA_COUNTERMEASURES
 
+/**
+ * \def MBEDTLS_AES_128_BIT_MASKED
+ *
+ * Requires MBEDTLS_AES_SCA_COUNTERMEASURES
```

Comment:
If this is required - please add a relevant check to check-config.h

Reply:
Fixed